src/MatchedLearnerApi/Program.cs

```diff
@@ -3,6 +3,7 @@
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
+using MatchedLearnerApi.Extensions;
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
```

Comment:
why are we using nlog?

Reply:
Not sure about this, it's worth double checking with Carl / Tom